tests/update_reference_data.sh

```diff
@@ -29,9 +29,15 @@ copyToReferenceDir () {
 }
 
 declare -A tests
-# binary dirname casename [testname]
-# you only have to specify testname if it differs from dirname
-tests[spe1]="flow spe1 SPE1CASE1"
```

Comment:
Isn't this missing now?

Reply:
good catch.

Reply:
Yes - that was good; I have fixed that now.